companies/serializers.py

```
@@ -116,10 +153,19 @@ def get_report_type(self, obj):
 class RevenueCompositionSerializer(serializers.ModelSerializer):
     """매출 구성 Serializer"""
 
+    ratio = serializers.SerializerMethodField()
+
     class Meta:
         model = RevenueComposition
         fields = ["segment_name", "revenue", "ratio"]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)